manifests/artemis.pp

```diff
@@ -0,0 +1,50 @@
+# Configuration for Artemis
+#
+# @api private
+class candlepin::artemis {
+  assert_private()
+
+  file { $candlepin::broker_config_file:
+    ensure  => file,
+    content => template('candlepin/broker.xml.erb'),
+    mode    => '0640',
+    owner   => 'tomcat',
+    group   => 'tomcat',
+  }
+
+  file { "${candlepin::catalina_home}/conf/login.config":
+    ensure => file,
+    source => 'puppet:///modules/candlepin/tomcat/login.config',
+    mode   => '0640',
+    owner  => 'tomcat',
+    group  => 'tomcat',
+  }
+
+  file { "${candlepin::catalina_home}/conf/cert-users.properties":
+    ensure  => file,
+    content => template('candlepin/tomcat/cert-users.properties.erb'),
+    mode    => '0640',
+    owner   => 'tomcat',
+    group   => 'tomcat',
+  }
+
+  file { "${candlepin::catalina_home}/conf/cert-roles.properties":
+    ensure => file,
+    source => 'puppet:///modules/candlepin/tomcat/cert-roles.properties',
```

Comment:
```suggestion
    content => file('candlepin/tomcat/cert-roles.properties'),
```

Reply:
When I do this, the tests fail.

Reply:
Here is the output:

```
      Puppet::PreformattedError:
        Evaluation Error: Error while evaluating a Function Call, Could not find any files from candlepin/tomcat/cert-roles.properties (file: /home/ehelms/workspace/upstream/modules/puppet-candlepin/spec/fixtures/modules/candlepin/manifests/artemis.pp, line: 33, column: 16) on node war.eagle
```

Reply:
You named it `files/tomcat/certs-roles.properties` on the filesystem.